single_tech_samples/databricks/sample3_cluster_provisioning_and_data_access/01.upload-sample-data.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
+
+DEPLOYMENT_PREFIX=${DEPLOYMENT_PREFIX:-}
+AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID:-}
+AZURE_RESOURCE_GROUP_NAME=${AZURE_RESOURCE_GROUP_NAME:-}
+AZURE_RESOURCE_GROUP_LOCATION=${AZURE_RESOURCE_GROUP_LOCATION:-}
+
+if [[ -z "$DEPLOYMENT_PREFIX" ]]; then
+    echo "No deployment prefix [DEPLOYMENT_PREFIX] specified."
+    exit 1
+fi
+if [[ -z "$AZURE_SUBSCRIPTION_ID" ]]; then
+    echo "No Azure subscription id [AZURE_SUBSCRIPTION_ID] specified."
+    exit 1
+fi
+if [[ -z "$AZURE_RESOURCE_GROUP_NAME" ]]; then
+    echo "No Azure resource group [AZURE_RESOURCE_GROUP_NAME] specified."
+    exit 1
+fi
+if [[ -z "$AZURE_RESOURCE_GROUP_LOCATION" ]]; then
+    echo "No Azure resource group [AZURE_RESOURCE_GROUP_LOCATION] specified."
+    exit 1
+fi
+
+# Variables
+
+storageAccountName="${DEPLOYMENT_PREFIX}asa01"
+storageAccountContainerName="sample-container"
+sampleDataFile="sample-data.us-population.json"
+
+# Create a storage container for file uploads
+storageKeys=$(az storage account keys list --resource-group "$AZURE_RESOURCE_GROUP_NAME" --account-name "$storageAccountName")
+storageAccountKey=$(echo "$storageKeys" | jq -r '.[0].value')
+containerExists=$(az storage container exists --account-name "$storageAccountName" --account-key "$storageAccountKey" --name "$storageAccountContainerName" | jq -r '.exists')
+if [[ "$containerExists" == "false" ]]; then
+    echo "Container $storageAccountContainerName does not existing and will be created."
+    az storage container create --account-name "$storageAccountName" --account-key "$storageAccountKey" --name "$storageAccountContainerName" 
+fi
+echo "Uploading file ./$sampleDataFile"
+az storage blob upload --account-name "$storageAccountName" --account-key "$storageAccountKey" --container-name "$storageAccountContainerName" \
+    --file "./$sampleDataFile" --name "$sampleDataFile"
+
+# Assign Storage Blob Data Contributor role to the current account
+storageAccountID=$(az storage account show --name "$storageAccountName" --query id --output tsv)
+assignee=$(az account show --query user.name --output tsv)
+az role assignment create --assignee "$assignee" --role "Storage Blob Data Contributor" --scope "$storageAccountID"
```

Comment:
Forgot to mention this one ;)
```suggestion
az role assignment create --assignee "$assignee" --role "Storage Blob Data Contributor" --scope "$storageAccountID" --output none
```

Reply:
Updated in the new script.